edit.php

```diff
@@ -31,12 +26,12 @@
 {
 	ImmersiveFormTemplate::AddImmersiveDialog('An error occurred', IMMERSIVE_ERROR, 'You can only edit your own plugins.', $Template);
 	$Template->SetRefresh('showplugin.php?id=' . $_GET['id']);
-	return;	
+	return;
 }
 
 if (isset($_POST['DeleteConfirmed' . $_GET['id']]))
 {
-	GitHubAPI::DeleteRepositoryUpdateHook($_GET['id'], $Query['UpdateHookID']);	
+	GitHubAPI::DeleteRepositoryUpdateHook($_GET['id'], $Query['UpdateHookID']);
 	$SQLLink->query('DELETE FROM PluginData WHERE RepositoryID = %i', $_GET['id']);
```

Comment:
MeekroDB should avoid injection attacks. I don't know why this project uses it over PDO, they're both about the same complexity, but it shouldn't be vulnerable to injection.